dw-al2-base-ami/dw-base-ami-install.sh

```diff
@@ -40,6 +40,10 @@ sed -i.bak -e 's/repo_upgrade: security/repo_upgrade: none/' \
 
 yum install -y python-pip gcc yum-plugin-remove-with-leaves sudo
```

Comment:
this still installs `python-pip` from `python2`.  Is that still needed?

Reply:
Both Python 2 and Python3 and pip2 and pip3 are needed.

Reply:
```suggestion
yum install -y python-pip gcc
```

OK, then I suggest the above as:

```
    amazon-ebs: Package yum-plugin-remove-with-leaves-1.1.31-46.amzn2.0.1.noarch already installed and latest version
    amazon-ebs: Package sudo-1.8.23-4.amzn2.2.x86_64 already installed and latest version
```
in the PR log.

Reply:
If we're blocking access in the proxy, then I'd imagine not.